src/main/java/org/spongepowered/api/data/key/Keys.java

```diff
@@ -865,6 +960,12 @@
      */
     public static final Key<Value<Boolean>> SUSPENDED = KeyFactory.fake("SUSPENDED");
 
+    /**
+     * Represents the {@link Key} for representing the scoreboard tags applied
+     * to an entity.
```

Comment:
Could you describe what a scoreboard tag is? Perhaps link it even to a wiki article?